.circleci/config.yml

```diff
@@ -679,28 +679,28 @@ workflows:
       - centos7-llvm-391-release
       - ubuntu-18.04-system-llvm-debug
       - ubuntu-18.04-system-llvm-release
+      - cross-llvm-700-debug-arm
+      - cross-llvm-700-release-arm
       - cross-llvm-600-debug-arm
       - cross-llvm-600-release-arm
       - cross-llvm-501-debug-arm
       - cross-llvm-501-release-arm
-      - cross-llvm-401-debug-arm
-      - cross-llvm-401-release-arm
       - cross-llvm-391-debug-arm
       - cross-llvm-391-release-arm
+      - cross-llvm-700-debug-armhf
+      - cross-llvm-700-release-armhf
       - cross-llvm-600-debug-armhf
       - cross-llvm-600-release-armhf
       - cross-llvm-501-debug-armhf
       - cross-llvm-501-release-armhf
-      - cross-llvm-401-debug-armhf
-      - cross-llvm-401-release-armhf
       - cross-llvm-391-debug-armhf
       - cross-llvm-391-release-armhf
+#      - cross-llvm-700-debug-aarch64
```

Comment:
Is there are reason the 600 versions are off @dipinhora ? Should they still be off? Should LLVM 7 be off?

Reply:
If we figure out the reason, and they remain off, we should leave a comment here in the configuration describing why.

Reply:
@SeanTAllen They're commented out/off because they fail (I left them there for easy re-enabling once things were fixed). There are equivalent jobs enabled in Travis CI in the `Allowed Failures` section in case they magically get fixed.